tests/integration/replication.tcl

```diff
@@ -994,3 +994,37 @@ start_server {tags {"repl external:skip"}} {
         }
     }
 }
+
+test {slave can handle EINTR if use diskless load} {
```

Comment:
let's avoid using the "s" word in new code, or at least in test names which are printed to the terminal.
```suggestion
test {replica can handle EINTR if use diskless load} {
```